cypress/integration/roamingTest.js

```diff
@@ -1,16 +1,16 @@
-import RoumingPage from '../../pages/RoamingPage'
+import RoamingPage from '../../pages/RoamingPage'
 import * as data from '../../test_data/roaming_data'
 
-describe(" Should verify tariffs in 'Rouming' page", ()=>{
+describe("Verification the titles of roaming tariffs", ()=>{
     
     // https://app.qase.io/case/CYAQA-47
     // https://app.qase.io/case/CYAQA-48
-    it("Should verify world-travel tariffs ", ()=>{
-        RoumingPage.navigateByUrl(data.roamingUrl);
+    it("verify titles of roaming tariffs", ()=>{
+        RoamingPage.openUrl(data.roamingUrl);
         cy.url().should('include', '/roaming/world-travel/')
```

Comment:
Here you can check not part of the URL, but full URL.
`cy.url().should('eq', 'https://.......')`